src/lib/isPostalCode.js

```diff
@@ -38,12 +44,13 @@ const patterns = {
   RU: sixDigit,
   SA: fiveDigit,
   SE: /^\d{3}\s?\d{2}$/,
+  SI: fourDigit,
   SK: /^\d{3}\s?\d{2}$/,
   TN: fourDigit,
   TW: /^\d{3}(\d{2})?$/,
   US: /^\d{5}(-\d{4})?$/,
   ZA: fourDigit,
-  ZM: fiveDigit,
+  ZM: fiveDigit
```

Comment:
The lint is failing because of the missing trailing comma here. Can you add it back in?